server/src/cloud/board.ts

```diff
@@ -301,6 +302,9 @@ export const initializeBoardFunctions = () => {
       }
       board.set("voting", request.board.voting);
     }
+    if (request.board.showNotesOfOtherUsers !== undefined) {
```

Comment:
You could use `!=` to check for both null and undefined

Reply:
Perfect. Thank you.

Reply:
Perfect. Thank you.